webapp/src/routes/EditProfile/styles.js

```diff
@@ -27,6 +27,17 @@ export default (theme) => ({
       color: theme.palette.secondary.onSecondaryMediumEmphasizedText
     }
   },
+  mobile: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    minHeight: 'calc(100vh - 60px)',
+    paddingTop: '20px',
+    [theme.breakpoints.up('md')]: {
+      paddingLeft: '1%',
+      paddingRight: '1%',
```

Comment:
padding: "0 1%",

Take a look at the https://www.w3schools.com/css/css_padding.asp shorthand

Reply:
Should i remove the padding? @JustinCast

Reply:
No, I meant that maybe you can use the shorthand. So instead of using `paddingLeft: '1%'` and `paddingRight: '1%'`, you can use `padding: "0 1%"`

Reply:
oh I see ok gonna do that thank you!